### PR TITLE
Test for upsert semantics in SQLite

### PR DESCRIPTION
**Pull request by Pchelolo**
Test for https://github.com/wikimedia/restbase-mod-table-sqlite/pull/31

### CODE REVIEW

File: test/functional/types.js [hunk 1]
```diff
@@ -218,6 +220,73 @@ describe('Types', function() {
...
+            .then(function(response){
+                deepEqual(response, {status:201});
+                return router.request({
+                    uri: '/restbase.cassandra.test.local/sys/table/typeTable/',
+                    method: 'put',
+                    body: {
+                        table: "typeTable",
+                        attributes: {
+                            string: 'string_override',
+                            int: '48',
```
Comment (d00rman) [inline test/functional/types.js]: Lose the quotes?

**Comment by d00rman** [comment]
Have you verified the cassandra back-end module with this change? (I trust you have, but let's make sure we won't brake something :P)

**Comment by Pchelolo** [comment]
@d00rman cassandra pass this new test.

**Comment by d00rman** [comment]
Published as [v0.1.7](https://github.com/wikimedia/restbase-mod-table-spec/tree/v0.1.7)

**Pull request merged**